hazelcast/src/main/java/com/hazelcast/nio/serialization/PortableReader.java

```diff
@@ -24,6 +24,8 @@
 /**
  * Provides a mean of reading portable fields from a binary in form of java primitives
  * arrays of java primitives , nested portable fields and array of portable fields.
+ * <p/>
+ * fieldName used in most of the methods may contain nested fields, like body.brain.iq
```

Comment:
what about this: PortableReader support nested paths. For example `<code>body.brain.iq</code>` is a valid field name.